app/controllers/centers_controller.rb

```diff
@@ -0,0 +1,30 @@
+class CentersController < ApplicationController
+	def index
+		@centers = Center.all
+	end
+
+	def show
+	end
+
+	def new
+		@center = Center.new
+	end
+
+	def edit
+	end
+
+	def create
+    @center = Center.new(center_params)
+
+    respond_to do |format|
```

Comment:
we don't need to respond to json requests. So it can be written without the `respond_to` and `format.` So it would look roughly like

```
if @center.save
  redirect_to ...
else
  render ...
end
```